vaultSDK/lib/udContext.dart

```diff
@@ -0,0 +1,50 @@
+import 'dart:async';
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:flutter/services.dart';
+
+import 'vdkLib.dart';
+import 'udError.dart';
+
+class UdContext {
+  static const MethodChannel _channel = const MethodChannel('udContext');
+
+  static Future<String> get platformVersion async {
+    final String version = await _channel.invokeMethod('getPlatformVersion');
+    return version;
+  }
+
+  // Call these functions in flutter widgets
+  static vdkError connect(Pointer<IntPtr> udContext, email, password,
+      [url = 'https://udstream.euclideon.com', appName = 'Pointfluent']) {
+    final uUrl = Utf8.toUtf8(url);
+    final uAppName = Utf8.toUtf8(appName);
+    final uEmail = Utf8.toUtf8(email);
+    final uPassword = Utf8.toUtf8(password);
```

Comment:
All four of these will leak memory unless you explicitly free that memory. The FFI package provides the `free` function for this.

Reply:
Fixed in e5c0ab9